var/spack/repos/builtin/packages/vim/package.py

```diff
@@ -90,6 +97,17 @@ def install(self, spec, prefix):
         else:
             configure_args.append("--enable-rubyinterp=dynamic")
 
+        if '+lua' in spec:
+            configure_args.append("--enable-luainterp=yes")
+            configure_args.append("--with-lua-prefix=%s" % spec['lua'].prefix)
+        else:
+            configure_args.append("--enable-luainterp=dynamic")
```

Comment:
What does `dynamic` mean here? Does it mean that it builds if it finds it and doesn't if it can't find it? If so, we don't want this non-deterministic behavior. Might be better to go with `no` for Lua and Perl.